rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -138,3 +138,93 @@
           }} // pubcloud slave
         }} // if
       }} // cit node
+
+- job:
+    name: JJB-artifacts-apt
+    node: ArtifactBuilder2
+    concurrent: false
+    build-discarder:
+      daysToKeep: 30
+    triggers:
+      - timed: "H 4 * * *"
+    parameters:
+      - pr-params
```

Comment:
As mentioned by the lintjob this macro doesn't exist, it can be added to params.yml or you can remove the line as ghprb is not listed in triggers.